packages/components/src/templates/dialog.stories.ts

```diff
@@ -0,0 +1,150 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Dialog',
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    design: {
+      type: 'figma',
+      url: ''
+    }
+  }
+};
+
+/**
+ *
+ */
+
+export const Default = {
+  name: 'Dialog with Button Group',
+  parameters: {
+    viewport: {
+      defaultViewport: 'iphone6'
+    }
+  },
+  render: () => html`
+    <div class="h-[40vh]">
+      <sd-dialog id="dialog" open>
+        <h4 slot="headline" class="sd-headline sd-headline--size-3xl">Cancel your subscription?</h4>
+        <p class="sd-paragraph">Your subscription will be canceled immediately. You can resubscribe at any time.</p>
+        <div slot="footer" class="flex justify-end w-full space-x-2">
+          <sd-button variant="secondary" size="lg" type="button"> Maybe later </sd-button>
+          <sd-button variant="primary" size="lg" type="button"> Cancel </sd-button>
+        </div>
+      </sd-dialog>
+    </div>
+  `
+};
+
+export const Scrollable = {
+  name: 'Dialog with scrollable content',
```

Comment:
How about the mobile screen that shows up automatically when the viewport is smaller than 414px.
It has smaller headline and paddings and the buttons are streched and vertically stacked.

Could we show that in the storybook template somehow?
I cannot switch to a mobile view in the templates.

Reply:
Pls increase the frame height to sth like 750px height and reduce the height of the dialog itself so that there is minium padding-top and padding-bottom (48px)
![Bildschirmfoto 2024-08-28 um 10 50 45](https://github.com/user-attachments/assets/0e150147-aca5-40ac-bd31-5fafff9d27fb)

Reply:
See previous comment regarding responsive behaviour.
I've updated the frame size and as best as possible.